app/src/main/res/layout/screen_teleconsult_share_prescription.xml

```diff
@@ -112,58 +109,63 @@
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
           android:layout_marginTop="@dimen/spacing_8"
+          android:nestedScrollingEnabled="false"
           app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
           app:layout_constraintBottom_toTopOf="@+id/sharePrescriptionMedicinesDivider"
           app:layout_constraintTop_toBottomOf="@+id/medicinesTextView" />
 
         <View
           android:id="@+id/sharePrescriptionMedicinesDivider"
-          style="@style/Clinic.V2.ListSeparator"
-          android:layout_gravity="bottom"
+          android:layout_width="match_parent"
+          android:layout_height="@dimen/spacing_1"
```

Comment:
We are using `ListSeparator` in some places and height spacing as 1dp in some places? Are we going to remove `ListSeparator` later?

Reply:
`ListSeparator` was a older implementation, that style has margin set to it by default. During material themeing changes I will try to create custom `DividerView` that would simplify this.